config/vector.yaml

```
@@ -37,7 +37,9 @@ sinks:
     type: "console"
     inputs: ["parse_logs"]
     encoding:
-      codec: "pretty_json"
+      codec: "json"
+      json:
+        use_pretty_json: true
```

```suggestion
        pretty: true
```

I think we can shorten this up a bit.